src/Classes/Helicoptere.php

```diff
@@ -0,0 +1,135 @@
+<?php
+
+class Helicoptere {
+
+    //coordonnees en x y z
+    public $x,$y,$z;
+    //direction vers laquelle est tournee l'helico
+    public $direction;
+
+    function __construct($posx,$posy) {
+        $this->$x=$posx;
+        $this->$y=$posy;
+        $this->$direction=0;
+    }
+
+    function seDeplacer($distance) {
+        $angle=($direction % 360);
+        switch ($angle) {
```

Comment:
This is hardly readable. Some tips to make it better:
- Use constants instead of degrees (STRAIGHT, BACKWARD, LEFT, RIGHT, DIAGONAL_STRAIGHT ...)
- Factorize cases that look a bit alike in their own methods